7_react/react1/week3/vite-project/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "vite-project",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "unfetch": "^5.0.0"
```

Comment:
`unfetch` is unused and could be removed. If you've looked into it, can you explain what this package would help you with and how it could have been used? 😄

Reply:
Hi
Thank you so much for your review. I was trying different methods for fetching, and at last, I chose the current one I have with .then. I couldn't figure out how it works and I got some errors 😅